pkgs/applications/misc/keepassx/community.nix

```diff
@@ -67,8 +69,8 @@ stdenv.mkDerivation rec {
   doCheck = true;
   checkPhase = ''
     export LC_ALL="en_US.UTF-8"
-    make test ARGS+="-E testgui --output-on-failure"
   '';
+    # make test ARGS+="-E testgui --output-on-failure"
```

Comment:
You don't have to comment this out. The checkPhase will still be ran.

Just set `doCheck = false;` and add a comment with why it can't be ran.
(that helps with the next person who may want to enable them)